home/.chezmoidata.yaml

```diff
@@ -1716,10 +1716,10 @@ softwarePlugins:
       - github/gh-net
       - mislav/gh-cp
   intellij-idea-ce:
-    cmd: bash -c 'idea.sh installPlugins "{PLUGIN}"'
+    cmd: bash -c '/snap/intellij-idea-community/current/bin/idea.sh installPlugins "{PLUGIN}"'
```

Comment:
Hey, can you keep this as `idea.sh` and create a symlink to the appropriate location in ~/.local/bin/idea.sh --- this will be better because there are a handful of different places the idea.sh might be